src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine{
+
+    @Override
+    public void doWork() {
+        System.out.println("Truck started its work ⚙.");

```

Comment:
The message should be more informative and indicate that it's a Truck that is working. Consider specifying the type of machine in the message, for example: `Truck is doing its work`.